src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/domain/goal/GoalBuilder.kt

```diff
@@ -0,0 +1,34 @@
+package uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.goal
+
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.goal.GoalCategory.PERSONAL_DEVELOPMENT
+import uk.gov.justice.digital.hmpps.educationandworkplanapi.domain.goal.GoalStatus.ACTIVE
+import java.time.Instant
+import java.time.LocalDate
+import java.util.UUID
+
+fun aValidGoal(
+  reference: UUID = UUID.randomUUID(),
+  title: String = "Improve communication skills",
+  reviewDate: LocalDate = LocalDate.now().plusMonths(6),
+  category: GoalCategory = PERSONAL_DEVELOPMENT,
+  steps: List<Step> = mutableListOf(aValidStep(), anotherValidStep()),
+  status: GoalStatus = ACTIVE,
+  notes: String? = "Chris would like to improve his listening skills, not just his verbal communication",
+  createdBy: String = "",
```

Comment:
I'm not sure what to set createdBy or updatedBy to yet, but we need to create a separate story to handle this.